core/src/main/java/dev/streamx/cli/ExceptionHandler.java

```diff
@@ -16,12 +16,15 @@ public class ExceptionHandler implements IExecutionExceptionHandler {
   @Override
   public int handleExecutionException(Exception ex, CommandLine cmd,
       ParseResult parseResult) {
-
-    cmd.getErr().println(cmd.getColorScheme().errorText(ex.getMessage()));
-    log.error("Execution exception occurred.", ex);
+    Throwable throwable = ex;
```

Comment:
a better name than throwable would be exceptionCause. 
also handle the cause that e.getCause() returns null (assign inital exception)